test/e2e/dataplane/service_discovery.go

```diff
@@ -42,4 +53,78 @@ func RunServiceDiscoveryTest(f *framework.Framework) {
 
 	Expect(err).NotTo(HaveOccurred())
 	Expect(stdout).To(ContainSubstring("Welcome to nginx!"))
+
+	By(fmt.Sprintf("Executing dig to verify Nginx service is discoverable and service IP in cluster %q is returned", framework.ClusterC))
```

Comment:
This isn't needed since an appropriate message is printed in `verifyClusterIpWithDig`.

Reply:
Done.